noisepagecontrol/exploratory_worker/scripts/stop_exploratory_cluster.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+##################################  NOTE  ##################################
+#   This script tears down the exploratory postgres cluster
+#   Script should be run as postgres user;
+#       $ sudo -u postgres scripts/stop_production_cluster.sh
+###############################################################################
+
+##############################  Configurations  ##############################
+
+# This specifies where the postgres binaries are (initdb, pg_ctl, etc.)
+# Common paths:
+#       Ubuntu: /usr/lib/postgresql/14/bin
+#       OSX:    /usr/local/bin
+POSTGRES_BIN_DIR="/usr/lib/postgresql/14/bin"
+
+# This specifies where the postgres cluster should reside
+#   Ubuntu: /var/lib/postgresql/14
+POSTGRES_ROOT="/var/lib/postgresql/14"
+
+###############################################################################
+
+# TODO Tim: pass in CLUSTER_DIR path
+CLUSTER_DIR="${POSTGRES_ROOT}/exploratory"
```

Comment:
Pending, should query db to find data dir (check get_data_dir.py in primary_executor)